src/abstract/ZoweTreeProvider.ts

```diff
@@ -80,6 +81,8 @@ export class ZoweTreeProvider {
      * @param isOpen the intended state of the the tree view provider, true or false
      */
     public async flipState(element: IZoweTreeNode, isOpen: boolean = false) {
+        element.collapsibleState = isOpen ? vscode.TreeItemCollapsibleState.Expanded : vscode.TreeItemCollapsibleState.Collapsed;
```

Comment:
Why have you added this?. The collapsible state is controlled by vscode and this is reacting to it being set. 
Although I agree with what you are setting is correct it feels like something that isn't broken (unless there are Theia issues?)

Reply:
By setting this property I eliminate necessity to pass this flag to generator of icons, I just want to rely on internal properties of the node. Hope it makes sense.